src/app/Notifications/NotificationCenter.tsx

```diff
@@ -37,35 +37,79 @@
  */
 import * as React from 'react';
 import { Dropdown, DropdownItem, DropdownPosition, KebabToggle,
-  NotificationDrawer, NotificationDrawerBody, NotificationDrawerHeader,
+  NotificationDrawer, NotificationDrawerBody, NotificationDrawerGroup, NotificationDrawerGroupList, NotificationDrawerHeader,
   NotificationDrawerList, NotificationDrawerListItem,
   NotificationDrawerListItemBody, NotificationDrawerListItemHeader, Text,
   TextVariants } from '@patternfly/react-core';
-import { Notification, NotificationsContext } from './Notifications';
+import { Notification, NotificationDrawerCategory, NotificationsContext } from './Notifications';
+import { combineLatest } from 'rxjs';
 
 export interface NotificationCenterProps {
   onClose: () => void;
 }
 
 export const NotificationCenter: React.FunctionComponent<NotificationCenterProps> = props => {
   const context = React.useContext(NotificationsContext);
-  const [notifications, setNotifications] = React.useState([] as Notification[]);
-  const [unreadNotificationsCount, setUnreadNotificationsCount] = React.useState(0);
+  const [totalUnreadNotificationsCount, setTotalUnreadNotificationsCount] = React.useState(0);
   const [isHeaderDropdownOpen, setHeaderDropdownOpen] = React.useState(false);
+  const PROBLEMS_CATEGORY_IDX = 2;
+  const [drawerCategories, setDrawerCategories] = React.useState([
+    {title: "Completed Actions", isExpanded: true, notifications: [] as Notification[], unreadCount: 0},
+    {title: "Network Info", isExpanded: false, notifications: [] as Notification[], unreadCount: 0},
+    {title: "Problems", isExpanded: false, notifications: [] as Notification[], unreadCount: 0}
+  ] as NotificationDrawerCategory[]);
+  const unreadProblemsCount = drawerCategories[PROBLEMS_CATEGORY_IDX].unreadCount;
```

Comment:
Won't this always initialize to 0?

Reply:
Removed this line after replacing the useEffect dependency on line 110 from 'unreadProblemsCount` to `drawerCategories`